### DISCUSSION TITLE
Help me with WalletConnectV2 on unity mobile platforms.

### DISCUSSION THREAD

**Discussion by angryera** [WalletConnect/WalletConnectUnity#134] [Q&A] [answered]
Hi WalletConnect Team,

I hope you're doing well. I'm currently working on a game that's available on WebGL and mobile platforms, and I've been using WalletConnect for integration. Previously, I had a smooth experience with WalletConnectV1, especially in terms of opening deep links for mobile wallets. However, I've recently upgraded to WalletConnectV2 and I'm encountering some challenges.

Could you please guide me on how to open deep links for mobile wallets using WalletConnectV2? Any advice or instructions would be greatly appreciated.

Thank you for your assistance!

Best regards.

**Comment by skibitsky** [comment, marked as answer]
Hi @angryera,

I'd recommend you to try our new WalletConnect Modal package. You can find docs [here](https://docs.walletconnect.com/advanced/walletconnectmodal/about?platform=unity).

Currently there's one more step necessary to enable WebGL support, read [more on WebGL here](https://github.com/WalletConnect/WalletConnectUnity#webgl-usage).
